Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:21-bookworm AS base
```

Comment:
Might be worth picking the alpine LTS version?

```suggestion
FROM node:20-alpine AS base
```

Reply:
do we wanna go bleeding edge yolo and go to node:22-alpine?

Reply:
Saw this comment and made a [fix to setup-app](https://github.com/safeinsights/setup-app/pull/2) to use the `alpine` variant for LTS there too :sweat_smile: